.github/workflows/closed-issue-message.yml

```diff
@@ -1,4 +1,4 @@
-name: Closed Issue Message
+name: "Closed Issue Message"
```

Comment:
don't believe these are needed. It's idiomatic in yaml to leave it the way it was

Reply:
I changed it because I thought that was why it was not showing the actual name in the Actions tab but showing `.github/workflows/closed-issue-message.yml ` instead. But I see that we have other workflows that don't use quotes for the name and it's displaying correctly. Not sure what the issue is. I'll remove the quotes.